src/Octoshift/Commands/GrantMigratorRole/GrantMigratorRoleCommandArgs.cs

```diff
@@ -23,5 +24,10 @@ public override void Validate(OctoLogger log)
         {
             throw new OctoshiftCliException("Actor type must be either TEAM or USER.");
         }
+
+        if (!string.IsNullOrEmpty(GhesApiUrl) && !string.IsNullOrEmpty(TargetApiUrl))
```

Comment:
nit:
we do have a `HasValue()` extension method for a string:
```suggestion
        if (GhesApiUrl.HasValue() && TargetApiUrl.HasValue())
```